server/src/test/java/io/crate/lucene/NumericEqQueryTest.java

```diff
@@ -68,4 +128,64 @@ public void test_uses_binary_encoded_range_queries_for_large_numeric() throws Ex
         assertThat(query).isInstanceOf(PointRangeQuery.class);
         assertThat(query.toString()).isEqualTo("y:[-99999999999999999999999999999999999999 TO 274679983711617676]");
     }
+
+    @Test
+    public void test_large_numeric_with_different_precision() {
```

Comment:
I think it would be enough to use one test but with an random reference (e.g. `String col = randomBoolean() ? "x" : "y"`)